.github/workflows/github-action.yml

```diff
@@ -0,0 +1,25 @@
+name: Automate dumptruck release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v1
+        - name: Login to DockerHub Registry
+          uses: docker/login-action@v1
+          with:
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
+            password: ${{ secrets.DOCKERHUB_PASSWORD }}
+        - name: Build Docker image
```

Comment:
t: there are actions available e.g. https://github.com/docker/build-push-action that would implement it elegantly